dotnet/src/Planners/Planners.OpenAI/Stepwise/FunctionCallingStepwisePlanner.cs

```diff
@@ -75,7 +80,11 @@ public sealed class FunctionCallingStepwisePlanner
         // Create and invoke a kernel function to generate the initial plan
         var initialPlan = await this.GeneratePlanAsync(question, clonedKernel, logger, cancellationToken).ConfigureAwait(false);
```

Comment:
The `initialPlan` appears to be unused in the case of a `chatHistoryForSteps` being provided. Since that step is making a call to the LLM it would be good to not waste the token/cycles.

Reply:
Thanks, updated.